### PR TITLE
add custom roles

### PR DESCRIPTION
**Pull request by Talits**

### CODE REVIEW

**Comment by Talits** [comment]
[[Agendor] Criar ansible com nossas personalizações para Kubernetes/KOPS](https://trello.com/c/LUN4Bjae/186-agendor-criar-ansible-com-nossas-personaliza%C3%A7%C3%B5es-para-kubernetes-kops)

File: installer/kops/playbooks/lambda-prom/config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tags:
+    - always
+  roles:
+    - init
+
+- name: Open ports
```
Comment (caruccio) [inline installer/kops/playbooks/lambda-prom/config.yml]: What ports?

File: installer/kops/roles/firewall/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+vpc_id: ""
+security_groupname: ""
+sg_destiny: ""
```
Comment (caruccio) [inline installer/kops/roles/firewall/defaults/main.yml]: rename vars:
```
security_groupname -> securitygroup_name
sg_destiny -> target_securitygroup
```

File: installer/kops/roles/lambda-prom/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+aws_account_id: ""
+pagerdutykey: ""
+webhook: ""
+
```
Comment (caruccio) [inline installer/kops/roles/lambda-prom/defaults/main.yml]: All those 3 vars already exist on getupengine.env. Check other `installer/$FLAVOR/deploy` for reference.

**Comment by caruccio** [comment]
revisado
--
Mateus Caruccio / Master of Puppets
GetupCloud.com
We make the infrastructure invisible
Gartner Cool Vendor 2017


Em seg, 19 de nov de 2018 às 15:57, Talita Bernardes Pereira <
notifications@github.com> escreveu:

File: installer/kops/roles/firewall/defaults/main.yml (hunk 2 above)
Comment (Talits) [reply installer/kops/roles/firewall/defaults/main.yml]: ok

File: installer/kops/roles/lambda-prom/defaults/main.yml (hunk 3 above)
Comment (Talits) [reply installer/kops/roles/lambda-prom/defaults/main.yml]: ok

**Pull request merged**